UnityGLTF/Assets/UnityGLTF/Runtime/Scripts/SceneImporter/ImporterTextures.cs

```diff
@@ -358,7 +375,15 @@ TextureWrapMode UnityWrapMode(GLTF.Schema.WrapMode gltfWrapMode)
 				if (!UnityEditor.AssetDatabase.Contains(source))
 #endif
 				{
-					var unityTexture = Object.Instantiate(source);
+					Texture2D unityTexture;
+					if (!source.isReadable)
+					{
+						unityTexture = new Texture2D(source.width, source.height, source.format, source.mipmapCount, isLinear);
+						Graphics.CopyTexture(source, unityTexture);
+					}
+					else
```

Comment:
When we create new textures here should we ensure the name matches the one we're copying from?

Reply:
will be set on line 392